src/Shared/PackageManagers/NuGetProjectManager.cs

```diff
@@ -221,9 +248,10 @@ private async Task UpdateVersionMetadata(PackageMetadata metadata, NuGetPackageV
             }
 
             string nameLowercase = packageVersionPackageVersionMetadata.Name.ToLowerInvariant();
```

Comment:
This doesn't appear to be a chance you introduced, but this also doesn't seem to be needed and causes the metadata to differ from the actual query used.